docusaurus/docs/iOS/guides/importance-of-synchronize.md

```diff
@@ -0,0 +1,36 @@
+---
+title: The Importance of Synchronize
+---
+
+## About `synchronize()`
+
+ Controllers are lightweight objects and they only fetch data when needed. When a controller is created, it doesn't fetch local or remote data until it needs it.
+
+`synchronize()` makes sure StreamChat's local database and backend is in sync. It queries the backend for the latest state of the Channel and updates the database. In addition, `synchronize()` call starts actually obversing the Channel for changes, so you will start getting live updates of the changes to the Channel, including it's messages.
+
+If you only need the local data, you can just access it after creation, like so:
+
+```swift
+let channelController = chatClient.channelController(for: ChannelId(type: .messaging, id: "general"))
+let channel = channelController.channel
+let messages = channelController.messages
+ ```
+
+In addition, you don't need to call `synchronize()` to be able to call actions on the channel, such as `freezeChannel`:
```

Comment:
This is only true for channels with id's passed. If one uses `createChannelWithMembers:`, they need to call `synchronize` to be able to call actions on the channel